components/DataView.vue

```diff
@@ -19,6 +19,15 @@
       <div class="DataView-CardText">
         <slot />
       </div>
+      <div v-if="this.$slots.dataTable" class="DataView-Details">
+        <details v-if="canvas">
+          <summary class="DataView-DetailsSummary" @click="toggleDetails">{{
+            $t('テーブルを表示')
```

Comment:
```suggestion
            $t('一覧表')
```
トリガーの `<summary>` 要素のテキストは、開いていても閉じていても整合性が取れるように、アクションを含まないテキスト、例えば「一覧表」などにしていただきたいですー。

Reply:
@oti 
レビューありがとうございます！

文言についてはこちらの issue https://github.com/tokyo-metropolitan-gov/covid19/issues/2068#issuecomment-604926637 で検討していただいたのですが、こちらでも「一覧表」や「一覧表を開く」などの案があり、テーブルやアクションを含まないほうがいいという意見もありました。

ですが、文言については運営の方々の意見を伺った上で調整するということでしたので、返答をまってから反映することになっていました。

@ocean3fish 
アコーディオンの文言については、運営の方の調整後に反映するということで変わりないでしょうか？
もしこの段階で変更できるのであれば、この PR 内で対応させていただきます。